pom.xml

```diff
@@ -14,7 +14,7 @@
    <parent>
       <groupId>org.eclipse.hawkbit</groupId>
       <artifactId>hawkbit-parent</artifactId>
-      <version>0.2.4</version>
+      <version>0.3.0-SNAPSHOT</version>
```

Comment:
Shouldn't you wait for 0.3.0M1?

Reply:
Sure, as soon as I get Nexus to cooperate